api/api.proto

```diff
@@ -1606,7 +1606,9 @@ service OpenStorageMigrate {
       };
     }
 
-  // Inspect the status of migration operation
+  // Status for migration operation
+  // if status request is empty, status for all migration
```

Comment:
This is markdown, and this would look weird in the output to https://libopenstorage.github.io. It should be:

```
// Status for migration operation.
// If status request is empty, status for all migration operation will be returned.
```

I added a `.` and capitalized the second sentence.